pkg/git/gogit/checkout_test.go

```diff
@@ -359,6 +371,374 @@ func TestCheckoutTagSemVer_Checkout(t *testing.T) {
 	}
 }
 
+// Test_KeyTypes assures support for the different types of keys
+// for SSH Authentication supported by Flux.
+func Test_KeyTypes(t *testing.T) {
+	tests := []struct {
+		name    string
```

Comment:
`name` is unused below.

Reply:
Nice catch @darkowlzz, just fixed it. PTAL.